jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,654 @@
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\"merge\": {}},\n",
+    "        {\"clip\": {}}\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print the response object\n",
+    "---\n",
+    "Retrieve the response object as text to print out to check if there are errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"_links\":{\"_self\":\"https://api.planet.com/compute/ops/orders/v2/af23033f-bc81-4515-a41f-7ad78ce63ddf\"},\"created_on\":\"2023-04-04T21:05:08.898Z\",\"delivery\":{\"google_cloud_storage\":{\"bucket\":\"devrel-notebooks\",\"credentials\":\"<REDACTED>\",\"path_prefix\":\"basemaps-to-cloud/\"}},\"error_hints\":[],\"id\":\"af23033f-bc81-4515-a41f-7ad78ce63ddf\",\"last_message\":\"Preparing order\",\"last_modified\":\"2023-04-04T21:05:08.898Z\",\"name\":\"basemap order with geometry\",\"order_type\":\"partial\",\"products\":[{\"geometry\":{\"coordinates\":[[[4.607406,52.353994],[4.680005,52.353994],[4.680005,52.395523],[4.607406,52.395523],[4.607406,52.353994]]],\"type\":\"Polygon\"},\"mosaic_name\":\"global_monthly_2022_01_mosaic\"}],\"source_type\":\"basemaps\",\"state\":\"queued\",\"tools\":[{\"merge\":{}},{\"clip\":{}}]}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "ORDERS_API_URL = 'https://api.planet.com/compute/ops/orders/v2'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "param_req = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "    order_params), auth=auth, headers={'content-type': 'application/json'})\n",
+    "\n",
+    "print(param_req.text)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Place the order\n",
+    "---\n",
+    "The following code places the order."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def place_order(order_params, auth):\n",
+    "    '''Place an order and return the order URL\n",
+    "\n",
+    "    Parameters: order_params (dict): order parameters\n",
+    "                auth (requests.auth.HTTPBasicAuth): authentication\n",
+    "    Return (str): URL of the order\n",
+    "    '''\n",
+    "    response = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "        order_params), auth=auth, headers=headers)\n",
+    "    order_id = response.json()['id']\n",
+    "    order_url = ORDERS_API_URL + '/' + order_id\n",
+    "    return order_url"
```

Comment:
You already placed the order on line 190 above, you shouldn't place another one here.
Either remove line 190, or remove this block.
If you keep this block, update it to use session + json, per the guidance on line 190